README.md

```diff
@@ -56,7 +56,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Thanks to the reviewers
+- Thanks to [Cindy Shin](https://github.com/Ukaypromise/) whose design gotten from behance was used as an inspiration for this project.
```

Comment:
- [ ] Please deploy your project to GitHub pages and the link to readme file 👍🏼

Reply:
Thanks for the professional review @Rustamxon7 . I have add a link to the to the deployed site  in the README.md file in the final-feature-branch.